src/main/scala/uk/gov/hmrc/initprototype/HerokuConfiguration.scala

```diff
@@ -33,7 +33,7 @@ class HerokuConfiguration {
   val jobTimeout: Duration                          = Duration(config.getInt("heroku.jobTimeoutMs"), MILLISECONDS)
   val connTimeoutMs: Int                            = config.getInt("heroku.connTimeoutMs")
   val readTimeoutMs: Int                            = config.getInt("heroku.readTimeoutMs")
-  val administratorEmails: Seq[String]              = config.getStringList("heroku.administratorEmails").asScala
+  val administratorEmails: Seq[String]              = config.getStringList("heroku.administratorEmails").asScala.toSeq
   val periodAfterWhichAppConsideredInactive: Period = Period.ofDays(84)
   val herokuAppsToKeepRunning: Set[String]          = config.getStringList("heroku.appsToKeepRunning").asScala.toSet
```

Comment:
looks like these 3 + `jobTimeout` are redundant now 💥

Reply:
oh, as @oscarduignan said, most of these `Heroku...` classes aren't being used in production code any more, so they and their tests can probably go